### DISCUSSION TITLE
没有IPV6了

### DISCUSSION THREAD

**Discussion by machine-doll** [immortalwrt/immortalwrt#773] [Q&A] [answered]
之前一直是有IPv6的，更新到21.02.1就没有了。之前不知道到底是什么版本，但是内核是5.10的，应该是Snapshot版吧  
尝试使用电脑拨号，一开始也没有拿到IPv6，怀疑是运营商的问题。但是后来知道其实是Win11的Bug，如果用Ubuntu拨号就可以正常拿到IPv6  
尝试过修改接口的参数，也没有用。之前一直都可以直接拿到IPv6，并不需要什么修改  
怀疑是这几次更新引入了bug，导致无法使用IPv6  
系统是在[官网](https://downloads.immortalwrt.org/releases/21.02.1/targets/x86/64/)上下载的，还试了其他几个版本，但是都不行

**Comment by 1715173329** [comment, marked as answer]
我们默认不集成 ipv6 支持，请您自行在 软件包 中安装 ipv6helper。

**Reply by machine-doll** [reply]
谢谢，按照[这篇文章](https://doc.openwrt.cc/1-General/5-IPV6-Guide/#:~:text=%E5%9C%A8%20OpenWrt%20%E7%9A%84%20SSH%20%E6%88%96%20TTYD%20%E4%B8%AD%E6%89%A7%E8%A1%8C%20ipv6-helper,can%20help%20you%20install%20IPV6%20modules%20on%20OpenWrt.)弄好了

**Reply by 1715173329** [reply]
我们的 ipv6helper 是集成在软件包中的，您可以直接通过 opkg 或 luci 界面安装。
